.env.integration-test

```diff
@@ -0,0 +1,16 @@
+SERVER_PORT=3000
+SERVER_HOST=0.0.0.0
+
+DATABASE_URL=postgres://user:password@localhost:5438/mydatabase
+DATABASE_USER=user
+DATABASE_PASSWORD=password
+DATABASE_DB=mydatabase
+# test credentials for integration test
+EXCHANGE_API_URL=https://openexchangerates.org/api/latest.json?app_id=5efbcc4908bf449293137c5a83c1f7b2
```

Comment:
Is that some app_id for testing?)

I wonder if that is safe to publish your own api keys to github)

Reply:
Yes, it's id for testing )